tst/EnergyPlus/unit/PlantCondLoopOperation.unit.cc

```diff
@@ -0,0 +1,770 @@
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+// EnergyPlus::Standalone ERV Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <DataPlant.hh>
+#include <EnergyPlus/PlantCondLoopOperation.hh>
+
+#include "Fixtures/EnergyPlusFixture.hh"
+
+using namespace EnergyPlus;
+
+class DistributePlantLoadTest : public EnergyPlusFixture
+{
+
+public:
+    static void SetUpTestCase()
+    {
+        EnergyPlusFixture::SetUpTestCase(); // Sets up the base fixture
+    }
+    static void TearDownTestCase()
+    {
+    }
+
+    virtual void SetUp()
+    {
+        EnergyPlusFixture::SetUp(); // Sets up individual test cases.
+
+                                    // unit test for plant equipment list load distribution
+                                    // set up one plantloop side with 1 branches, 12 components
+                                    // using 12 components here to test going beyond the old idd limit of 10 pieces of equipment
+        DataPlant::PlantLoop.allocate(1);
+        DataPlant::PlantLoop(1).OpScheme.allocate(1);
+        DataPlant::PlantLoop(1).OpScheme(1).EquipList.allocate(1);
+        auto &thisEquipList(DataPlant::PlantLoop(1).OpScheme(1).EquipList(1));
+        thisEquipList.NumComps = 12;
+        thisEquipList.Comp.allocate(thisEquipList.NumComps);
+
+        DataPlant::PlantLoop(1).LoopSide.allocate(1);
+        DataPlant::PlantLoop(1).LoopSide(1).Branch.allocate(1);
+        DataPlant::PlantLoop(1).LoopSide(1).Branch(1).Comp.allocate(thisEquipList.NumComps);
+        auto &thisBranch(DataPlant::PlantLoop(1).LoopSide(1).Branch(1));
+
+        for (int compNum = 1; compNum <= DataPlant::PlantLoop(1).OpScheme(1).EquipList(1).NumComps; ++compNum) {
+            // set up equipment list data
+            thisEquipList.Comp(compNum).CompNumPtr = compNum;
+
+            thisEquipList.Comp(compNum).BranchNumPtr = 1;
+
+            // set up individual component data - start with 12 equal size, all available
+            thisBranch.Comp(compNum).Available = true;
+            thisBranch.Comp(compNum).OptLoad = 90.0;
+            thisBranch.Comp(compNum).MaxLoad = 100.0;
+            thisBranch.Comp(compNum).MinLoad = 0.0;
+            thisBranch.Comp(compNum).MyLoad = 0.0;
+        }
+    }
+
+    virtual void ResetLoads()
+    {
+        // reset loads
+        auto &thisBranch(DataPlant::PlantLoop(1).LoopSide(1).Branch(1));
+        for (int compNum = 1; compNum <= DataPlant::PlantLoop(1).OpScheme(1).EquipList(1).NumComps; ++compNum) {
+            thisBranch.Comp(compNum).MyLoad = 0.0;
+        }
+    }
+    virtual void TearDown()
+    {
+        EnergyPlusFixture::TearDown(); // Remember to tear down the base fixture after cleaning up derived fixture!
+
+    }
```

Comment:
Since your derived fixture doesn't actually need to do any special tear-down, you might not need to override TearDown here.  No biggy, and I could be wrong anyway, but since you are just calling the base class TearDown method, and not doing anything else here, it should be the same behavior to just not have your own overriding method.